backend/gn_module_monitoring/command/utils.py

```diff
@@ -116,6 +116,137 @@ def process_available_permissions(module_code, session):
         )
 
 
+def clean_existing_permissions(module_code, session):
+    try:
+        module = session.scalars(select(TModules).where(TModules.module_code == module_code)).one()
+    except NoResultFound:
+        print(f"Le module {module_code} n'est pas présent")
+        return
```

Comment:
**Suggestion:** Consider handling exceptions more specifically to avoid catching unexpected errors. [best practice]
```suggestion
try:
    module = session.scalars(select(TModules).where(TModules.module_code == module_code)).one()
except NoResultFound as e:
    print(f"Le module {module_code} n'est pas présent: {str(e)}")
    return
except Exception as e:
    print(f"An unexpected error occurred: {str(e)}")
    return
```